src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java

```diff
@@ -213,13 +217,17 @@
                       CONFIG_DISPLAY_MQ_QUEUE);
 
         config.define(CONFIG_NAME_MQ_USER_NAME, Type.STRING, null, Importance.MEDIUM,
-                      CONFIG_DOCUMENTATION_MQ_USER_NAME, CONFIG_GROUP_MQ, 7, Width.MEDIUM,
-                      CONFIG_DISPLAY_MQ_USER_NAME);
+                      CONFIG_DOCUMENTATION_MQ_CSP, CONFIG_GROUP_MQ, 7, Width.MEDIUM,
+                      CONFIG_DISPLAY_MQ_CSP);
 
         config.define(CONFIG_NAME_MQ_PASSWORD, Type.PASSWORD, null, Importance.MEDIUM,
                       CONFIG_DOCUMENTATION_MQ_PASSWORD, CONFIG_GROUP_MQ, 8, Width.MEDIUM,
                       CONFIG_DISPLAY_MQ_PASSWORD);
 
+        config.define(CONFIG_NAME_MQ_CSP, Type.STRING, true, Importance.MEDIUM,
+                CONFIG_DOCUMENTATION_MQ_USER_NAME, CONFIG_GROUP_MQ, 7, Width.MEDIUM,
+                CONFIG_DISPLAY_MQ_USER_NAME);
+
```

Comment:
Looks like CONFIG_DOCUMENTATION_MQ_* and CONFIG_DISPLAY_MQ_* variables has been mixed up on lines 220-221 + 228-229

Reply:
Done